packages/components/src/InlineLabel/InlineLabel.tsx

```diff
@@ -55,6 +67,15 @@ export function InlineLabel({
       <Typography element="span" size={sizeMapper[size]} textCase="uppercase">
         {children}
       </Typography>
+      {dismissable && (
+        <ButtonDismiss
+          onClick={() => {
+            onDismiss && onDismiss();
+          }}
+          ariaLabel={dismissAriaLabel || ""}
```

Comment:
you could put that empty string as a default so you won't have to null check it.

![image](https://user-images.githubusercontent.com/15986172/131402773-ad3b894f-cf89-45a3-8d71-925794505f75.png)

Also, thoughts on putting "Remove" as the default?